### PR TITLE
Hookup test_config get route

### PR DESCRIPTION
**Pull request by dcelik**
This PR relies on code, specifically convert_video_to_frames, from the video-playing [branch](https://github.com/santosfamilyfoundation/SantosGUI/tree/video-playing) and as such is in an untested state. After #52 is merged into this branch it can be tested and reviewed.

This PR adds the following functionality:
 * getTestConfig method in cloud_api uses the testConfig GET route to download the configuration videos
 * [EDITED] videos obtained from testConfig are turned into frames, and played in the GUI

### CODE REVIEW

File: application/app.py [hunk 1]
```diff
@@ -121,6 +120,21 @@ def test_feature(self):
...
+
+    def getFeatureVideo(self):
+        project_path = get_project_path()
+        api.getTestConfig('feature', get_identifier(), project_path)
+
+        images_folder = os.path.join(project_path, 'feature_video', 'images')
+        video_path - os.path.join(project_path, 'feature_video')
+
+        #TODO: The code is on an un-PR'ed branch
+        convert_video_to_frames(video_path, images_folder, 'feature_images', 'JPEG')
```
Comment (runnersaw) [inline application/app.py]: I'd use `'jpg'`

**Review by runnersaw — COMMENTED** [review]
Initial review

**Review by jabb1123 — APPROVED** [review]

**Comment by dcelik** [comment]
See #61 for info about changes made for testing

**Review by runnersaw — APPROVED** [review]
:ship: :it:

**Pull request merged**